pkg/kubelet/dockershim/docker_service.go

```diff
@@ -130,7 +132,7 @@ func NewDockerService(client dockertools.DockerInterface, seccompProfileRoot str
 	cniPlugins = append(cniPlugins, kubenet.NewPlugin(pluginSettings.PluginBinDir))
 	netHost := &dockerNetworkHost{
 		pluginSettings.LegacyRuntimeHost,
-		&namespaceGetter{ds},
```

Comment:
I understand. 
I feel like getting NetNS and PortMapping are mandatory at this point. And there is no other use case for a PortMappingGetter for now. The NoopPortMappingGetter should be able to help with composing the network host right? Or I can add a PortMappingGetter interface.